localstack/services/generic_proxy.py

```diff
@@ -172,6 +172,18 @@ def forward(self, method):
         if 'localhost.atlassian.io' in forward_headers.get('Host'):
             forward_headers['host'] = 'localhost'
 
+        x_forwarded_for = forward_headers.get('X-Forwarded-For')
```

Comment:
Since this method is already pretty lengthy, can we put this code into a small utility method in this class to improve readability?